apps/src/code-studio/pd/permission.js

```diff
@@ -0,0 +1,12 @@
+export default class Permission {
+  constructor() {
+    this.isAdmin = this.hasPermission('workshop_admin');
```

Comment:
Good point. I'll update to `isWorkshopAdmin` Btw, this change didn't add that, but merely wrapped it in a class instead of duplicating the isAdmin logic in many components.